portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +30,20 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+  private ArrayList<String> comments = new ArrayList<String>();
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Garrett!</h1>");
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = request.getParameter("text-input");
+    comments.add(text);
+    Entity commentEntity = new Entity("Comment");
```

Comment:
FYI. Using "Comment" as your key will give each comment you create the same key. You can use KeyFactory to make unique keys for each comment. 

https://cloud.google.com/appengine/docs/standard/java/javadoc/com/google/appengine/api/datastore/KeyFactory#createKey-java.lang.String,%20long-